tools/auth/get_id_token.go

```diff
@@ -0,0 +1,64 @@
+package auth
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+// Key for verify custom token must be saved in PROJECTAPIKEY environment variable
+// example: PROJECTAPIKEY=<projectApiKey> go run get_id_token.go <customToken>
+
+// getIDToken verify customToken and return idToken
+func getIDToken(customToken string, projectAPIKey string) interface{} {
+	requestBody, err := json.Marshal(map[string]string{
+		"token":             customToken,
+		"returnSecureToken": "true",
+	})
+
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	resp, err := http.Post(
+		"https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyCustomToken?key="+projectAPIKey,
+		"application/json",
+		bytes.NewBuffer(requestBody))
+
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	var result map[string]interface{}
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	idToken := result["idToken"]
+	if idToken == nil {
+		log.Fatalln("Invalid projectAPIKey or customToken.")
```

Comment:
I'd rather return empty interface (or empty string, see the comment above) after Fatalf, just to be sure when somebody changes this from Fatalf to Errorf that the logic does not continue.